docs/hub-deployment-guide/cloud-accounts/new-gcp-project.md

```diff
@@ -22,6 +22,10 @@
    Make sure the correct project is selected while enabling these!
    ```
 
+7. If we have direct access to the billing account used, we must set up
+   [billing export to BigQuery](https://cloud.google.com/billing/docs/how-to/export-data-bigquery#setup).
```

Comment:
Can we actually provide steps if not automation as this  isscriptable with gcloud (with beta for the billing part) see this comment

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/issues/292#issuecomment-1447516383

https://cloud.google.com/sdk/gcloud/reference/beta/billing/projects/link

 there doesn't look to be a terraform provider yet but it should be possible 
https://github.com/hashicorp/terraform-provider-google/issues/4848

Reply:
@pnasrat I think for this PR, I'm going to just document how to do this instead of automate this.

Reply:
@pnasrat opened https://github.com/2i2c-org/infrastructure/issues/2442